aws/table_aws_eks_node_group.go

```diff
@@ -0,0 +1,257 @@
+package aws
+
+import (
+	"context"
+
+	"github.com/turbot/steampipe-plugin-sdk/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/plugin/transform"
+
+	"github.com/aws/aws-sdk-go/service/eks"
+)
+
+func tableAwsEksNodeGroup(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_eks_node_group",
+		Description: "AWS Elastic Kubernetes Service Node Group",
+		Get: &plugin.GetConfig{
+			KeyColumns: plugin.AllColumns([]string{"name", "cluster_name"}),
+			Hydrate:    getEksNodeGroup,
+		},
+		List: &plugin.ListConfig{
+			ParentHydrate: listEksClusters,
+			Hydrate:       listEksNodeGroup,
+		},
+		GetMatrixItem: BuildRegionList,
+		Columns: awsRegionalColumns([]*plugin.Column{
+			{
+				Name:        "name",
+				Description: "The name associated with an Amazon EKS managed node group.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("NodegroupName"),
+			},
+			{
+				Name:        "node_group_arn",
+				Description: "The Amazon Resource Name (ARN) of the cluster.",
+				Type:        proto.ColumnType_STRING,
+				Hydrate:     getEksNodeGroup,
+				Transform:   transform.FromField("nodegroupArn"),
```

Comment:
Is `nodegroupArn` the correct field name ? Please check if we are getting the correctvalues for both list and get calls.